lib/Swagger/src/ContentEntity/Api/ContentFeedApi.php

```diff
@@ -99,6 +101,128 @@ public function setApiClient(\Swagger\Client\ApiClient $apiClient)
+     * @param bool $include_root_relations  (optional, default to true)
+     * @param bool $show_connection_details  (optional, default to false)
+     * @param bool $restrict_universe  (optional, default to true)
+     * @param string[] $content_order  (optional)
+     * @return Array of \Swagger\Client\ContentEntity\Models\MixedRelatedContent, HTTP status code, HTTP response headers (array of strings)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function getOrderedRelatedContentWithHttpInfo($wiki_id, $limit = null, $include_root_relations = null, $show_connection_details = null, $restrict_universe = null, $content_order = null)
+    {
+        // verify the required parameter 'wiki_id' is set
+        if ($wiki_id === null) {
+            throw new \InvalidArgumentException('Missing the required parameter $wiki_id when calling getOrderedRelatedContent');
+        }
+        // parse inputs
+        $resourcePath = "/content-feed/from-wiki-id/{wikiId}";
+        $httpBody = '';
+        $queryParams = array();
+        $headerParams = array();
+        $formParams = array();
+        $_header_accept = $this->apiClient->selectHeaderAccept(array('application/json'));
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
+        }
+        $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType(array());
+
+        // query params
+        if ($limit !== null) {
+            $queryParams['limit'] = $this->apiClient->getSerializer()->toQueryValue($limit);
+        }
+        // query params
+        if ($include_root_relations !== null) {
+            $queryParams['includeRootRelations'] = $this->apiClient->getSerializer()->toQueryValue($include_root_relations);
+        }
+        // query params
+        if ($show_connection_details !== null) {
+            $queryParams['showConnectionDetails'] = $this->apiClient->getSerializer()->toQueryValue($show_connection_details);
+        }
+        // query params
+        if ($restrict_universe !== null) {
+            $queryParams['restrictUniverse'] = $this->apiClient->getSerializer()->toQueryValue($restrict_universe);
+        }
+        // query params
+        if (is_array($content_order)) {
+            $content_order = $this->apiClient->getSerializer()->serializeCollection($content_order, 'multi', true);
+        }
+        if ($content_order !== null) {
+            $queryParams['contentOrder'] = $this->apiClient->getSerializer()->toQueryValue($content_order);
+        }
+        // path params
+        if ($wiki_id !== null) {
+            $resourcePath = str_replace(
+                "{" . "wikiId" . "}",
+                $this->apiClient->getSerializer()->toPathValue($wiki_id),
+                $resourcePath
+            );
+        }
+        // default format to json
+        $resourcePath = str_replace("{format}", "json", $resourcePath);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "{format}" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)